internal/policy/policy.go

```diff
@@ -201,6 +199,8 @@ func LoadStateForEntry(ctx context.Context, repo *git.Repository, entry *rsl.Ref
 		state.RootPublicKeys = append(state.RootPublicKeys, key)
 	}
 
+	// TODO: verify root from original state? We have consecutive verification
+	// in place elsewhere.
```

Comment:
I'm going to take this TODO out because right now, in full verification we know the root is verified from the start. This really applies in the context of workflows like https://github.com/gittuf/gittuf/issues/245.

Reply:
When we implement those "trusted till commit A, verify only from then" workflows, we need to consider if we want to bootstrap only the policy changes from the start, ignoring other RSL entries.